packages/plugin-conversation/src/process-image.browser.js

```diff
@@ -84,7 +84,7 @@ export default function processImage({file, thumbnailMaxWidth, thumbnailMaxHeigh
       const ctx = canvas.getContext(`2d`);
       orient(
         {
-          orientation: file.image.orientation,
+          orientation: file && file.image ? file.image.orientation : ``,
```

Comment:
file does exist. It is just the file.image does not exist which was causing the issue.